src/types/IConsoleWindow.hpp

```diff
@@ -18,7 +18,7 @@ Author(s):
 // copied typedef from uiautomationcore.h

```

Comment:
this file should be moved back to where it was. that'll also eliminate a bunch of the changes in the PR.

Reply:
(Just writing what we discussed in person):
- IConsoleWindow, WindowUiaProvider, and more files to come will be moved into this shared space
- They will be moved into a new shared LIB (name TBD) to keep Types independent of dependencies

Upcoming changes to ScreenInfoUiaProvider include:
- Moving it to a shared space (same LIB as above)
- Adding some pure virtual functions to abstract out the ServiceLocator and other things we don't need from ConHost